data-prepper-plugins/opensearch/README.md

```diff
@@ -118,7 +118,7 @@ Default is null.
 
 - `aws_sts_header_overrides`: An optional map of header overrides to make when assuming the IAM role for the sink plugin.
 
-- `insecure`: A boolean flag to turn off SSL certificate verification. If set to true, CA certificate verification will be turned off and insecure HTTP requests will be sent. Default to `false`.
+- `insecure`: A boolean flag to turn off SSL certificate verification. If set to true, CA certificate verification will be turned off and insecure HTTP requests will be sent. Setting this will override any `cert` configured. Default to `false`.
```

Comment:
I added the same documentation clarification on the *source* side as on the *sink* side. Peeking in the configuration code for source, it appears that `insecure: true` already overrides `cert`, so this is just a clarification.